### PR TITLE
Move from --rpc to --rest in docker-compose examples

### PR DESCRIPTION
**Pull request by hukkin**

### CODE REVIEW

**Review by tersec — APPROVED** [review]

File: docker/dist/binaries/docker-compose-example1.yml [hunk 1]
```diff
@@ -28,9 +28,9 @@ services:
       --log-level=info
       --tcp-port=9000
       --udp-port=9000
-      --rpc
-      --rpc-address=0.0.0.0
-      --rpc-port=9190
+      --rest
+      --rest-address=0.0.0.0
+      --rest-port=9190
```
Comment (etan-status) [inline docker/dist/binaries/docker-compose-example1.yml]:
```suggestion
      --rest-port=5052
```

Comment (hukkin) [reply docker/dist/binaries/docker-compose-example1.yml]: If we change this, then the port forwarding above needs to change too.

Comment (hukkin) [reply docker/dist/binaries/docker-compose-example1.yml]: Committed the change. Arguably the example made more sense when configuring to a non-default port, but maybe it was confusing to use the legacy RPC port...

Comment (etan-status) [reply docker/dist/binaries/docker-compose-example1.yml]: Well, previously, the example was using the default port (9190 for RPC), so using 5052 now just makes it consistent with how it used to be.

**Pull request merged**